src/main/xar-resources/data/author-reference/author-reference.xml

```diff
@@ -158,7 +161,8 @@ but does not because its too simple)</programlisting>
     <sect1 xml:id="tech">
         <title>Technical guidelines</title>
 
-        <para>More information about Docbook itself can be found on the <link condition="_blank" xlink:href="http://docbook.org/">official Docbook site</link>.</para>
+        <para>More information about Docbook itself can be found on the <link condition="_blank"
```

Comment:
If we really want to be picky, it's DocBook.

Reply:
i can do another run